### PR TITLE
Added renderAsArray, mathMin, and mathMax

### PR DESCRIPTION
**Pull request by chris-j-tang**
#199 
#113 
#114

### CODE REVIEW

File: src/Languages/Properties/MathProperties.ts [hunk 1]
```diff
@@ -10,10 +10,24 @@ export class MathProperties {
...
+    /**
+     * How to retrieve the lower of two numbers.
+     */
+    public min: NativeCallProperties;
+
+    /**
      * Any imports native math commands require.
      */
     public requiredImports: { [i: string]: string[] };
-
```
Comment (JoshuaKGoldberg) [inline src/Languages/Properties/MathProperties.ts]: Why delete this line?

File: src/Languages/Properties/NativeCallProperties.ts [hunk 2]
```diff
@@ -3,6 +3,11 @@
  */
 export enum NativeCallScope {
     /**
+     * Called as a (I don't know what to put here?)
```
Comment (JoshuaKGoldberg) [inline src/Languages/Properties/NativeCallProperties.ts]: Restating what I mentioned on Slack in response to your question:

> Good question. Comments are hard... What you're doing in code is calling a member of an array containing the parameters. Something similar to that?

File: test/integration/MathMax/math max.cs [hunk 3]
```diff
@@ -0,0 +1,5 @@
+-
+using System;
+
+Math.Max(foo, bar);
+-
```
Comment (JoshuaKGoldberg) [inline test/integration/MathMax/math max.cs]: These files should have an endline so there's no little red symbol at the end in the PR.

Added a mention to #185.

File: docs/Design/Numbers/Native Math.md [hunk 4]
```diff
@@ -23,6 +23,49 @@ variable : bar int -7
 math absolute : bar
 ```
 
+### `math floor`
+
+`math floor` `:` `name`
+
+Generates code to compute the largest integer <= the given number.
+
+## Usage
```
Comment (JoshuaKGoldberg) [inline docs/Design/Numbers/Native Math.md]: Could you actually switch the `## Usage` headings to `#### Usage`? Having fewer `#`s than the <code>### `math floor`</code> ones is an existing typo.

File: src/Languages/Properties/MathProperties.ts (hunk 1 above)
Comment (chris-j-tang) [inline src/Languages/Properties/MathProperties.ts]: Oh, I forgot to retain it when modifying the file. Adding it back.

**Comment by JoshuaKGoldberg** [comment]
:+1:

**Pull request merged**